axelor-account/src/main/java/com/axelor/apps/account/service/config/AccountConfigService.java

```diff
@@ -482,7 +463,8 @@ public String getReimbursementExportFolderPath(AccountConfig accountConfig)
           I18n.get(com.axelor.apps.base.exceptions.IExceptionMessage.EXCEPTION),
           accountConfig.getCompany().getName());
     }
-    return accountConfig.getReimbursementExportFolderPath();
+    String dataExportDir = appService.getDataExportDir();
+    return dataExportDir + File.separator + accountConfig.getReimbursementExportFolderPath();
```

Comment:
There is no need to do any change here as you can check the returned value is not used anywhere, this method is just used to throw exception if accountConfig.getReimbursementExportFolderPath() is null